ui/app/AppLayouts/Profile/views/ContactsView.qml

```diff
@@ -51,42 +51,39 @@ SettingsContentBase {
             input.placeholderText: qsTr("Search by a display name or chat key")
         }
 
-        TabBar {
+        StatusTabBar {
             id: contactsTabBar
             Layout.fillWidth: true
             Layout.leftMargin: Style.current.padding
             Layout.rightMargin: Style.current.padding
             Layout.topMargin: 2 * Style.current.padding
-            height: contactsBtn.height
-            background: Rectangle {
-                color: Style.current.transparent
-            }
+            
             StatusTabButton {
                 id: contactsBtn
-                addToWidth: Style.current.bigPadding
-                btnText: qsTr("Contacts")
+                width: implicitWidth
```

Comment:
What a reason for it? `implicitWidth` should be original size and if `width` not set it should use `implicitWidth`